packages/homepage/content/articles/codesandbox-ci-embeds/index.md

```diff
@@ -0,0 +1,102 @@
+
+## Here's how it works
+
+CodeSandbox CI is a GitHub app that auto-builds your library⁠ from pull
+requests. Whenever someone opens a new pull request, CodeSandbox CI builds a new
+version of your library. Those builds get posted to our registry, so you can
+test it in CodeSandbox or locally, and all without having to publish the build
+to npm⁠. Plus, if the original GitHub issue contains a link to a reproducible
+sandbox, then we'll automatically fork it using the new PR build. That means you
+can review it online with one click.
+
+## Used by the biggest names in OSS
+
+After a successful private beta, CodeSandbox CI is already in use by projects
+like React, Babel, Framer, Material UI, and others. And it's proving to be a
+real time-saver.
+
+**Try it out for yourself by
+[installing the GitHub app](https://github.com/apps/codesandbox)**.
+
+[![Framer tweet](./images/mattgperry_tweet_1181863225756934144.png)](https://twitter.com/mattgperry/status/1181863225756934144?s=20)
+
+## A new look for Embeds
+
+CodeSandbox CI is the latest way maintainers can use CodeSandbox to save time
+and improve experiences for users and contributors. But more than 4000 open
+source projects already use CodeSandbox embeds in their docs, blog posts, and
+websites—providing visitors with working examples that help explain how to use
+their creations.
+
+Embeds provide the ability to view a sandbox inline in your content, whether you
+want to show the code, the running app, or both at the same time. Our embeds use
+a specially designed version of CodeSandbox that's small in size but almost as
+powerful as the full thing, making for a great way to explain a coding concept
+or show off a demo.
+
+We want you to be able to use embeds anywhere you want. So based on feedback
+from our community, we've launched a new default look and feel for embeds that's
+minimal in design and looks great wherever you place it. With a subtle color
+palette and no branding, it doesn't take the focus away from your great content.
+
+Danny Ruchtie, Head of Design here at CodeSandbox, notes that with this release,
+_“we've focused on building something that would better blend in with its
+surroundings while maintaining all its powerful features. So there's less focus
+on the UI and more on the content. We also wanted to give users more control and
+improve the embed experience on mobile.”_
+
+See the difference for yourself:
+
+<iframe
+  src="https://codesandbox.io/embed/new?fontsize=14"
+  style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+  title="new"
+  allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media; usb"
+  sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"
+></iframe>
```

Comment:
This can just be the link to the sandbox and [`gatsby-remark-embedder`](https://github.com/MichaelDeBoey/gatsby-remark-embedder) will handle it too.

```md
https://codesandbox.io/s/new?fontsize=14
```